lib/Rex/Commands/SCM.pm

```diff
@@ -19,26 +19,31 @@ All these functions are not idempotent.
 =head1 SYNOPSIS
 
  use Rex::Commands::SCM;
- 
+
```

Comment:
Please add those leading spaces back. They are marking the code block and needed for correct POD formatting.